picard/formats/vorbis.py

```diff
@@ -30,72 +37,304 @@
 except ImportError:
     OggOpus = None
     with_opus = False
-from picard import config, log
-from picard.coverart.image import TagCoverArtImage, CoverArtImageError
-from picard.file import File
-from picard.formats.id3 import types_from_id3, image_type_as_id3_num
-from picard.metadata import Metadata
-from picard.util import encode_filename, sanitize_date
+
+import base64
+from os import path
+from time import strftime, gmtime, gmtime
 
 
 class VCommentFile(File):
 
-    """Generic VComment-based file."""
+    """
+    Generic VComment-based file.
+    Specifications:
+        Formal spec: http://www.xiph.org/vorbis/doc/v-comment.html
+        In the wild: http://lists.xiph.org/pipermail/vorbis-dev/attachments/20090716/e2db9203/attachment-0001.xls
+        See also: http://age.hobba.nl/audio/mirroredpages/ogg-tagging.html
+        See also: http://www.mediamonkey.com/wiki/index.php/WebHelp:About_Track_Properties/4.0
+    """
     _File = None
 
-    __translate = {
-        "musicbrainz_trackid": "musicbrainz_recordingid",
-        "musicbrainz_releasetrackid": "musicbrainz_trackid",
+    __load_tags = {
+        # The following are taken from official V-Comment spec. at:
+        # http://www.xiph.org/vorbis/doc/v-comment.html
+        "title": "title",
+        #"version": "~releasecomment",
+        "album": "album",
+        "tracknumber": "tracknumber",
+        "artist": "artist",
+        "performer": "performer",
+        "copyright": "copyright",
+        "license": "license",
+        "organization": "label",
+        #"description": "?",
+        "genre": "genre",
+        "date": "date",
+        "location": "recordinglocation",
+        "contact": "web_official_label",
+        "isrc": "isrc",
+
+        # The following are taken from field recommendations at:
+        # http://age.hobba.nl/audio/mirroredpages/ogg-tagging.html
+        "artist": "artist",
+        "publisher": "label",
+        "label": "label",
+        "discnumber": "discnumber",
+        "disc": "discnumber",
+        "barcode": "barcode",
```

Comment:
That tag is not mentioned on http://age.hobba.nl/audio/mirroredpages/ogg-tagging.html.